tests/API/XCCDF/unittests/Makefile.am

```diff
@@ -82,6 +82,8 @@ EXTRA_DIST += \
 	test_oval_without_definition.oval.xml \
 	test_oval_without_definition.sh \
 	test_oval_without_definition.xccdf.xml \
+	test_profile_selection_by_suffix.sh \
+	test_profile_selection_by_suffix.xccdf.xml \
 	test_remediate_perl.sh \
```

Comment:
Please, put `test_profile_selection_by_suffix_tailoring.xccdf.xml` here as well. Now it's causing `make distcheck` to fail.